README.md

```diff
@@ -60,16 +60,25 @@ This plugin aims to port commands from Bukkit's Essentials to MCServer, but also
 ###Use-permissions
 | Permission | Description |
 | ---------- | ----------- |
-| warp.createsign | Allows a player to create Warp Signs |
+| es.warpsign | Allows a player to create Warp Signs |
 | es.enchantsign | Allows a player to create Enchant Signs |
+| es.createportal | Allows a player to create a portal |
 | es.home.unlimited | Allows a player to have an unlimited amount of homes |
 
-###Warp Sign template
+###Warp Sign Template
 [Warp]  
-warpname
+Warpname
 
-###Enchant Sign template
+###Enchant Sign Template
 [Enchant]  
 EnchantmentID  
 EnchantmentLevel  
 RequiredXPLevelToEnchant
+
+###Portals
```

Comment:
I think these should be [AdditionalInformation tables in the `Info.lua` file](http://mc-server.xoft.cz/LuaAPI/InfoFile.html#AdditionalInformation)?